Scripts/script-JIRAPrintIssue.yml

```diff
@@ -0,0 +1,82 @@
+commonfields:
+  id: JIRAPrintIssue
+  version: -1
+name: JIRAPrintIssue
+script: |-
+  var response = executeCommand('jira-get-issue', { issueId: args.issueId });
+
+  if (!response || response.length === 0 || !response[0].Contents) {
+      return 'JIRA issue not found (' + args.issueId + ')';
```

Comment:
I ran it with random Id and it did not catch it
instead it throws `Cannot access member 'summary' of undefined` (probably from line 18)